projects/storefrontapp-e2e-cypress/package.json

```diff
@@ -14,7 +14,8 @@
     "cy:run:mobile": "cypress run --spec \"cypress/integration/mobile/**/*\"",
     "cy:run:mobile:ci": "cypress run --config-file cypress.ci.json --spec \"cypress/integration/mobile/**/*\"",
     "cy:run:regression": "cypress run --spec \"cypress/integration/regression/**/*\"",
-    "cy:run:regression:ci": "cypress run --config-file cypress.ci.json --spec \"cypress/integration/regression/**/*\""
+    "cy:run:regression:ci": "cypress run --config-file cypress.ci.json --spec \"cypress/integration/regression/**/*\"",
+    "cy:cds:run:vendor": "cypress run --spec \"cypress/integration/vendor/cds/**/*\""
```

Comment:
just putting it out there, we have this ticket that needs to be either done by argo or seewhy

https://github.com/SAP/spartacus/issues/7462

left a comment for them as well https://cxjira.sap.com/browse/ARGO-5987?focusedCommentId=1033193&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-1033193